### PR TITLE
Release Drafter: Inherit settings from Global configuration

### PR DESCRIPTION
**Pull request by oleg-nenashev**
# Description

Downstream of https://github.com/jenkinsci/.github/pull/1 . Uses the standard Release Drafter configuration, except some project-specific overrides.

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [x] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
@halkeye FYI the `internal` tag has different meaning in the Release Drafter config (I was following the Jenkins Core notation). We could simplify the config by changing `internal` to `skip-changelog` in this repo

**Comment by halkeye** [comment]
renamed to skip-changelog

**Comment by halkeye** [comment]
can you add a comment at the top of the page for the extends for like https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.yml that would be perfect

just makes it easier to track down later

**Comment by halkeye** [comment]
oh i can edit.

Updated and now merging

File: .github/release-drafter.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
+# from  https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.yml
 _extends: .github
 name-template: BlueOcean $NEXT_PATCH_VERSION
 tag-template: blueocean-parent-$NEXT_PATCH_VERSION
 exclude-labels:
```
Comment (oleg-nenashev) [inline .github/release-drafter.yml]: No need, it will take it from the global config now

Comment (halkeye) [reply .github/release-drafter.yml]: no need for the exclude labels? or no need for the comment?

**Pull request merged**